mysql-test/suite/innodb/t/innodb_buffer_pool_resize_mdev_27891.test

```diff
@@ -0,0 +1,30 @@
+--source include/have_innodb.inc
+
+SET @save_size=@@GLOBAL.innodb_buffer_pool_size;
+
+let $wait_timeout = 60;
+let $wait_condition =
+  SELECT SUBSTR(variable_value, 1, 31) = 'Completed resizing buffer pool.'
+  FROM information_schema.global_status
+  WHERE variable_name = 'INNODB_BUFFER_POOL_RESIZE_STATUS';
+
+--echo
+--echo MDEV-27891: Delayed SIGSEGV in InnoDB buffer pool resize
+--echo after or during DROP TABLE
+--echo
+
+select @@innodb_buffer_pool_chunk_size;
+CREATE TABLE t1 (a INT PRIMARY KEY) ENGINE=InnoDB;
+SET GLOBAL innodb_buffer_pool_size=200*1024*1024;
```

Comment:
This should probably be `big_test.inc`, or the requested size should be much smaller (if possible).

I would not include ticket numbers in test file names, but instead use descriptive file names and add ticket numbers in comments inside the file.

Reply:
its defiantly somewhere between 128 and 200M. 92M and below had a very low hit rate.